src/map/ai/helpers/pathfind.cpp

```diff
@@ -434,12 +435,9 @@ void CPathFind::StepTo(const position_t& pos, bool run)
         }
     }
 
-    m_POwner->loc.p.moving += (uint16)((0x36 * ((float)m_POwner->speed / 0x28)) - (0x14 * (mode - 1)));
+    m_POwner->loc.p.moving += speedChange ? 0x28 : 0x35;
```

Comment:
I'm not sure what this was all about, but quickly looking at some caps I was seeing a pretty consistent 53 move increase, with occasional 40s. Rarely I would see something +/- 1 or 2 (a couple 39, 51, 52, but mostly 53 and 40). 40 seemed to coincide with melee attacks/speed changes (range related) so that's how I ended up with this. I noticed that if I didn't decrease this during speed changes, the mob would teleport forward occasionally when the distance got too large (for instance, when kiting a tiger up to about 30', it would teleport to 20').